backend/Models/Property.cs

```diff
@@ -0,0 +1,43 @@
+using System;
```

Comment:
so clearly these models are incredibly not finalized, but I'd like to start working on the frontend/backend connectivity on Openshift sooner rather then later. Once that communication is working I'll worry about the exact structure of the request/response (hopefully the database will exist by then).

Reply:
See my other comments related to models.